ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -39,9 +39,11 @@ tests:
       OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
```

Comment:
@liqcui Is there any specific reason or discussion that lead to specifying this as new control plane instance type or you just want to update this to use m6i instances? 
Were we using m5.xlarge before?

Reply:
This change should by some failure of cluster-density-heavy, but it seems it's not relate with the instance type, restore to default now